video/image_writer.c

```diff
@@ -62,6 +75,9 @@ const struct m_opt_choice_alternatives mp_image_writer_formats[] = {
     {"webp", AV_CODEC_ID_WEBP},
 #if HAVE_JPEGXL
     {"jxl",  AV_CODEC_ID_JPEGXL},
+#endif
+#if HAVE_AVIF_MUXER
+    {"avif",  AV_CODEC_ID_AV1},
```

Comment:
Is there ever a case when you have the avif muxer but not AV_CODEC_ID_AV1? If not, that's fine, I just want to double check.

Reply:
The ffmpeg commit already mentions AV_CODEC_ID_AV1, so I think we're good.